components/mailbox/src/Mailbox.svelte

```diff
@@ -56,13 +56,15 @@
   export let query_string: string; // Allowed query parameter list https://developer.nylas.com/docs/api/#get/threads
   export let show_star: boolean;
   export let show_thread_checkbox: boolean;
+  export let with_composer: boolean;
```

Comment:
Why does this prop need to exist still? We can just dispatch the reply clicked events regardless of whether there's a composer. A user may want to use their own email composer or have something else happen when reply_to is clicked.

Reply:
Thinking more about it, the prop should be something like `show_reply_to`, since we don't want to show the reply buttons if users aren't going to do anything with them.